src/components/RadioGroup/RadioGroupItem.tsx

```diff
@@ -62,6 +62,9 @@ class RadioGroupItem extends AutoControlledComponent<Extendable<IRadioGroupItemP
     /** Additional CSS class name(s) to apply.  */
     className: PropTypes.string,
 
+    /** Shorthand for content of the radio item. */
+    content: customPropTypes.contentShorthand,
```

Comment:
`shorthand` test should be added for this prop

Reply:
but, in fact, this is not a shorthand.. seems that propType name is misleading - please, ignore the comment about tests necessary

Reply:
any changes you'd suggest?

Reply:
no-no, nothing - let me close this one, sorry for misleading you